include/z64audio.h

```diff
@@ -3,6 +3,8 @@
 
 #define AUDIO_MK_CMD(b0,b1,b2,b3) (_SHIFTL(b0, 24, 8) | _SHIFTL(b1, 16, 8) | _SHIFTL(b2, 8, 8) | _SHIFTL(b3, 0, 8))
 
+#include "audiothread_cmd.h"
```

Comment:
Feel like this should probably be in the cmd header since that's where it's most used.

Reply:
In the cmd header? Isn't `audiothead_cmd.h` the cmd header itself?

Reply:
Yes, the macro for constructing the command. It's not like it's a .inc.c.